tests/helpers/errors.py

```diff
@@ -4,31 +4,23 @@ def get_error_test_cases(errors):
 
 ERRORS = {
     'invalid-parameters': {
-        "parameter": {
-            "example": [
-                "The example field is required"
-            ]
-        },
         "type": "https://www.rev.ai/api/v1/errors/invalid-parameters",
-        "title": "Your request parameters didn't validate",
+        "title": "Your request parameters did not validate",
         "status": 400
     },
     'unauthorized': {
         "title": "Authorization has been denied for this request",
         "status": 401
     },
+    'forbidden': {
+        "status": 403,
+        "error": "HIPAA user is not allowed to use MediaUrl"
+    },
```

Comment:
I"m confused about this PR. Looks like we change tests only here. Also, we return 403 when we use wrong language or human transcriber.

Reply:
Yes, because we have test cases that today are not used in the test and are misleading. I'll make the 403 test more generic. The main point of this test is just that we raise the error if we get one back from the api.